microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -0,0 +1,627 @@
+---
+title: Working with local content in WebView2 apps
+description: Ways to work with local content in WebView2 apps, including navigating to a file URL, navigating to a string, virtual host name mapping, or the WebResourceRequested event.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 01/10/2023
+---
+# Working with local content in WebView2 apps
+
+In addition to loading remote content, content can also be loaded locally into WebView2.  There are several different approaches that can be used to load local content into a WebView2 control, including: 
```

Comment:
Nit redundancy: there are several different --> there are different approaches

Reply:
There are several approaches that can be used to